### ISSUE TITLE
Implement the postprocessing of the satellite images

### ISSUE THREAD

**Issue by esloch** [AlertaDengue/AlertaDengueCaptura#60]
> Once this is done a new issue can be opened to implement the postprocessing of the images to generate the data to be inserted in the table Clima_Satelite. This table schema may need to be redesigned, to conform with the data available. Issue #22 has an older proposal.

ref:
https://github.com/AlertaDengue/AlertaDengueCaptura/blob/master/downloader_app/examples.ipynb
